ci/assemble_full_jar.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set +e
+
+mv Upload-*/* .
+
+mkdir -p ./build/install/lib
+mkdir -p ./build/install/arm/lib
+mkdir ./build/install/lib64
+mkdir ./build/tiledb_jni/
+mkdir ./build/tiledb_jni/arm
+mkdir ./build/tiledb_jni/Release
+mkdir ./build/install/bin
+
+
+for arch in $(ls | grep .gz.tar)
+do
+tar -xvf $arch
+done
+
+mv binaries_*/* .
+
+# OSX
+mv libtiledb.dylib ./build/install/lib
+mv libtiledbjni.dylib ./build/tiledb_jni
+mv arm/libtiledb.dylib ./build/install/arm/lib
+mv arm/libtiledbjni.dylib ./build/tiledb_jni/arm
+
+# Linux
+cp libtiledb.so ./build/install/lib
```

Comment:
Not necessary to make the change in this PR, but at some point soon we should change the structure to use specific subdirectories for each os/architecture.